### PR TITLE
FH-3610 - FHC tool - Add GitHub links to issue tracker to the package…

### PR DESCRIPTION
**Pull request by camilamacedo86**
JIRA: https://issues.jboss.org/browse/FH-3610?filter=12328970

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,7 +75,7 @@
     "turbo-test-runner": "0.3.3"
   },
   "engines": {
-    "node": ">=0.10"
+    "node": ">=4.4"
```
Comment (wei-lee) [inline package.json]: why do we change this?

Comment (camilamacedo86) [inline package.json]: In the ReadMe[1] and consequently npm page is this information, so make sense change one of the both sides. Or we change the doc or in the code. What/Where do you think it should be better? Is really good we still supporting deprecated and so old versions of the node for this project? 

[1] - https://github.com/feedhenry/fh-fhc#node-version-

**Comment by camilamacedo86** [comment]
re-test

File: package.json (hunk 1 above)
Comment (wei-lee) [inline package.json]: oh, ok. If other places state that it requires 4.4 and above, then this is ok

**Pull request merged**